build.gradle.kts

```diff
@@ -76,16 +88,8 @@ allprojects {
     }
     tasks.withType<DokkaTask>().configureEach {
         dokkaSourceSets.configureEach {
-            reportUndocumented.set(false)
             skipDeprecated.set(true)
             jdkVersion.set(8)
-            perPackageOption {
-                matchingRegex.set("build\\.buf.*")
-                suppress.set(true)
-            }
-        }
-        if (name == "dokkaGfm") {
-            outputDirectory.set(project.file("${project.rootDir}/docs/3.x"))
```

Comment:
Not sure where this is coming from - we don't publish GFM anywhere so I've switched to the standard `dokkaHtml` format.